test/integration.test.ts

```diff
@@ -107,6 +108,8 @@ describe('Starting cardano-wallet (and its node)', () => {
             kind: 'byron',
             configurationDir: '' + process.env.BYRON_CONFIGS,
             network: byron.networks.mainnet,
+            delegationCertificate: path.join(dataRoot, 'node.cert'),
+            signingKey: path.join( dataRoot, 'node.key')
```

Comment:
If this config makes mainnet cardano-node fail, perhaps add a new test case which sets up a self node config.

Reply:
I just mocked up a self config and it's still failing, so will merge and ensure we have a green suite once the other test fixes are in place.